packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -238,6 +238,26 @@ export enum AlbScheme {
   INTERNET_FACING = 'internet-facing',
 }
 
+/**
+ * Helm chart options that can be set for AlbControllerChart
+ * To add any new supported values refer
+ * https://github.com/kubernetes-sigs/aws-load-balancer-controller/blob/main/helm/aws-load-balancer-controller/values.yaml
+ */
+export interface AlbControllerHelmChartOptions {
+
+  /**
+   * Enable or disable the AWS WAF add on for the ALB ingress controller.
+   * @default - no value defined for this helm chart option, so it will not be set in the helm chart values
+   */
+  readonly enableWafv2?: boolean;
+
+  /**
+   * Enable or disable the AWS WAFv2
```

Comment:
```suggestion
   * Enable or disable the AWS WAFv2.
```
minor change to re-trigger the codebuild